README.md

```diff
@@ -87,6 +87,21 @@ print(list(sorted(ng.optimizers.registry.keys())))
 
 The [optimization documentation](docs/optimization.md) contains more information on how to use several workers, take full control of the optimization through the `ask` and `tell` interface and some pieces of advice on how to choose the proper optimizer for your problem.
 
+## Multiobjective minimization with Nevergrad
+
+Let us minimize x1 and x2 (two objective functions) assuming that values above 2.5 are of no interest.
+```python
+import nevergrad as ng
+
+f = ng.functions.multiobjective_minimization([lambda x: x[0], lambda x: x[1]], bad_values=[2.5, 2.5])
+optimizer = ng.optimizers.CMA(instrumentation=f.instrumentation, budget=100)  # 2 is the dimension, 100 is the budget.
+recommendation = optimizer.optimize(f)
+
+
+# The function embeds its Pareto-front:
+print("My Pareto front:", f())
```

Comment:
given that the calls to `f` can be performed in another process by an executor, this actually won't always work :s we'll need to discuss this :s